Bottom: 3ac02ca0ee786e913d18ea9852678a2b6a8d5279
Top:    013de92444b555fb48f3ce5893b5ba7a95d668c8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 13:11:45 +0100

Refresh of hid-gtk-turn-on-defines-to-deb

---

diff --git a/configure.ac b/configure.ac
index 71ab1c4..141b5d2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -770,6 +770,8 @@ $GTKGLEXT_PKG_ERRORS])]
 	GTKGLEXT_VER=`$PKG_CONFIG gtkglext-1.0 --modversion`
 	fi
 
+	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+
 	;;
 
       gcode|nelma|png )
